src/main/java/org/spongepowered/api/service/economy/EconomyService.java

```diff
@@ -108,6 +109,20 @@
      */
     Optional<Account> getOrCreateAccount(String identifier);
 
+    /**
+     * Returns a {@link Stream} of all stored {@link UniqueAccount} objects.
+     *
+     * @return A {@link Stream} of all stored {@link UniqueAccount} objects.
+     */
+    Stream<UniqueAccount> uniqueAccounts();
```

Comment:
Add Stream and Collection getters.

I like Streams but we should not shoehorn people into doing them as a default. I made the same change for the CatalogRegistry.

Reply:
Done

Reply:
Maybe also use the same phrasing in the CatalogRegistry that I've seen in other libraries?

https://github.com/SpongePowered/SpongeAPI/blob/api-8/src/main/java/org/spongepowered/api/registry/CatalogRegistry.java#L96

Reply:
@ImMorpheus I meant the method names.

Reply:
Oops, done